.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "0.21.0"
```

Comment:
The login and admin ui versions are prefixed with `v` (unlike our other rocks), does it need to be included?

Reply:
no need for that, see  https://github.com/canonical/identity-platform-admin-ui/pull/530